app/full_text_search.py

```diff
@@ -244,9 +267,10 @@ def create_romanized_name_fields(romanize_method, **kwargs):
     for field_name in kwargs:
```

Comment:
You can write

    for field_name, field_value in kwargs.iteritems():

to get the value instead.